packages/widgetbook/lib/src/fields/color_field/color_space.dart

```diff
@@ -0,0 +1,5 @@
+enum ColorSpace {
+  rgba,
```

Comment:
Let's rename it to `rgb`, because the alpha component will be shared with all color space. For example, the usual `hex` color space does not have the alpha value by default.